vendor/modules.txt

```diff
@@ -93,8 +93,6 @@ github.com/gabriel-vasile/mimetype
 github.com/gabriel-vasile/mimetype/internal/charset
 github.com/gabriel-vasile/mimetype/internal/json
 github.com/gabriel-vasile/mimetype/internal/magic
-# github.com/golang/snappy v0.0.4
-## explicit
```

Comment:
weird is that automatically removed?

Reply:
running `make fmt lint` did that, I believe